docs/src/modules/utils/layoutStorageKey.tsx

```diff
@@ -0,0 +1,5 @@
+export const dataGridLayoutKeys = {
+  slots: 'dataGrid_apiPage_slots',
```

Comment:
Side note: I just realized that the slots section is missing the v7 data grid docs:

v6: https://mui.com/x/api/data-grid/data-grid/#slots
v7: https://next.mui.com/x/api/data-grid/data-grid/#slots

Any ideas on how to bring it back? 😅

Reply:
Yes, the slots are generated by detecting interfaces named `[componentName]Slots`.

You can override this behavior with `slotInterfaceName`

Here is the uscase for date pickers:
https://github.com/mui/mui-x/blob/master/scripts/buildApiDocs/pickersSettings/getComponentInfo.ts/#L26-L28

Reply:
I added the fix